### DISCUSSION TITLE
New user question.

### DISCUSSION THREAD

**Discussion by Leiipt** [NightscoutFoundation/xDrip#1923] [Q&A]
Hi, new user here.
Well new in all things xdrip, github and having diabetes.

My question is about the bluetooth requirement of the xdrip app. I have libre 1 the sensor I get from the hospital. 

Is there no way to get the sensory data into xdrip other than to have libre 2 or a bluetooth bridge? The data is already in my phone in the librelink app. Can't I just share the data or point to the file storing the data and get xdrip to read it?

The reason I found xdrip to be interessting is because of librelinks very limited way of accessing and view of the data.

With kind regards, a noob.

**Comment by meatychunks99** [comment]
IIRC if your phone is compatible with the librelink app, then you should be able to use the NFC of your phone with xDrip - though it's been a few years since we last used a Libre. Choose Libre as the source in xDrip and I believe there is then the further option to select the librelink app as a source. I think you can then scan your libre sensor with your phone to pass the readings directly to xDrip.

**Reply by Leiipt** [reply]
Thank you for the answer. Didn't know it worked that way I should have tested before asking.
I will get to it right away.

Once more thank you for the answer. :)
